github/Requester.pyi

```diff
@@ -17,7 +17,7 @@ class HTTPRequestsConnectionClass:
         strict: bool = ...,
         timeout: Optional[int] = ...,
         retry: Any = ...,
-        **kwargs: Dict[str, str]
+        **kwargs: str
```

Comment:
not `Any`?

Reply:
This is HTTP{,S}ConnectionClass, not search_* -- it only gets strings as keyword arguments. I was naughty and fixed it in the same commit, bad me.

Reply:
But isn't this a Dict?